examples/mqtt-service-basic-authentication/mqtt_service_basic_authentication.md

```diff
@@ -0,0 +1,18 @@
+# MQTT service - Basic authentication
+
+The `mqtt:Service` receives messages from the MQTT server using the `mqtt:Listener` via basic authentication. Basic authentication can be enabled by configuring the `username`, `password` in the `connectionConfig` of the `mqtt:ListenerConfiguration`. Use this to connect to an MQTT server secured with basic authentication.
+
+::: code mqtt_service_basic_authentication.bal :::
+
+## Prerequisites
+- Start a [MQTT broker](https://mqtt.org/software/) instance configured to use basic authentication.
+
+Run the program by executing the following command.
+
+::: out mqtt_service_basic_authentication.out :::
+
+>**Tip:** Run the MQTT client given in the [MQTT publisher - Basic authentication](/learn/by-example/mqtt-client-basic-authentication) example to produce some messages to the topic.
```

Comment:
```suggestion
>**Tip:** Run the MQTT client given in the [MQTT publisher - Basic authentication](/learn/by-example/mqtt-client-basic-authentication) example to produce some messages to the topic.
```
Should this be one of the steps (last step) rather than a tip? There should be a step if it is mandatory to do it when trying out the example.

Reply:
This is not mandatory. If the users wants to view any output, they can run the client sample